build-logic/jvm/src/main/kotlin/gradlebuild.strict-compile.gradle.kts

```diff
@@ -18,7 +18,7 @@ import gradlebuild.jvm.extension.StrictCompileExtension
 
 extensions.create<StrictCompileExtension>("strictCompile")
 
-val strictCompilerArgs = listOf("-Werror", "-Xlint:all", "-Xlint:-options", "-Xlint:-serial", "-Xlint:-classfile")
+val strictCompilerArgs = listOf("-Werror", "-Xlint:all", "-Xlint:-options", "-Xlint:-serial", "-Xlint:-classfile", "-Xlint:-try")
```

Comment:
If you were curious like me, this avoids 

`warning: [try] auto-closeable resource ignore is never referenced in body of corresponding try statement`

in code like:


```
try (EvaluationContext.ScopeContext ignore = openScope()) {
           // no reference to "ignore" here
}       
```